lib/app_profiler/railtie.rb

```diff
@@ -11,7 +11,12 @@ class Railtie < Rails::Railtie
       AppProfiler.logger = app.config.app_profiler.logger || Rails.logger
       AppProfiler.root = app.config.app_profiler.root || Rails.root
       AppProfiler.storage = app.config.app_profiler.storage || Storage::FileStorage
-      AppProfiler.viewer = app.config.app_profiler.viewer || Viewer::SpeedscopeViewer
+      if app.config.app_profiler.stackprof_viewer
```

Comment:
I changed this because the remote firefox viewer has a peer dependency we may or may not need. It depends which backend the developers uses. This way it won't be loaded until it is used.

Reply:
👍